setup/02-robot_syntax_rule.py

```diff
@@ -1,25 +1,66 @@
-from rflint.common import SuiteRule, TestRule, ERROR
+from rflint.common import SuiteRule, TestRule, ERROR, WARNING
+from static import normalize
 
 MUST_KEYWORDS = [
         "Open Browser",
         "Input Text",
-        "Page Should Contain"
+        "Input Password"
     ]
 
+LOGIN_KEYWORDS = [
+        "Click Element",
+        "Click Button",
+        "Submit Form"
+]
+
+VERIFY_KEYWORDS = [
+        "Page Should Contain",
+        "Page Should Contain Element",
+        "Title Should Be",
+        "Wait Until Page Contains",
+        "Wait Until Page Contains Element"
+]
+
 class TestCaseImplementation02(TestRule):
     severity = ERROR
 
     def apply(self, test):
+        default_message = "Did you find all keywords from SeleniumLibrary documentation?"
         report = False
         test_steps = []
         for step in test.steps:
             if len(step) > 1:
-                test_steps.append(step[1])
+                test_steps.append(normalize(step[1]))
+
         for keyword in MUST_KEYWORDS:
-            if not keyword.strip() in test_steps:
+            if not normalize(keyword) in test_steps:
                 report = True
         if report:
-            self.report(test, "Did you find all keywords from seleniumlibrary documentation?, expected: {}".format(", ".join(MUST_KEYWORDS)), test.linenumber)
+            self.report(test, default_message + ", expected: {}".format(", ".join(MUST_KEYWORDS)), test.linenumber)
+
+        for keyword in test_steps:
+            if normalize(keyword) in LOGIN_KEYWORDS:
+                break
+        else:
+            self.report(test, default_message + ", expected one of: {}".format(", ".join(LOGIN_KEYWORDS)), test.linenumber)
+
+        for keyword in test_steps:
+            if normalize(keyword) in VERIFY_KEYWORDS:
+                break
+        else:
+            self.report(test, default_message + ", expected one of: {}".format(", ".join(VERIFY_KEYWORDS)), test.linenumber)
+
+class TestCaseKeywordCases02(TestRule):
+    severity = WARNING
+
+    def apply(self, test):
+        default_message = "Did you find all keywords from seleniumlibrary documentation?"
```

Comment:
This default message is the same as in previous class, maybe move them to `MODULE_LEVEL_VARIABLE`?